src/context/context.py

```diff
@@ -167,6 +167,19 @@ def __graph_ql(self, query: str, query_variables: dict) -> Union[dict, None]:
             key = next(iter(result))
             data = result[key]
         except Exception as e:
+            if len(e.args) == 1:
```

Comment:
I'd say this should be a derived Exception-class instead of string literal matching. Is is possible to raise a more specific exception in the first place and catch that exception here?

Reply:
It is difficult to do it, as the exception already comes from gql library in a form of custom exception. Do you have any ideas of how to ovecome it?

Reply:
Is it then of type *GraphQLError* or anything like that at least?

Reply:
Why don't we adjust the `get_organization`, `get_project` etc. functions and remove the return of `None`. Instead we could raise and error that the context is invalid. Then there is no need to catch the exception in `__graph_ql` and we can handle it in the `get_organizaation` etc. functions with a more precise error message.